client/src/modules/lead/EditLead/EditLeadContent/EditLeadTabs/EditLeadTabs.tsx

```diff
@@ -69,27 +73,59 @@ class EditLeadTabs extends React.Component<Props, State> {
   }
 
   public saveNote = (noteText: string) => {
-    const { editLead } = this.props;
-    const note = {
-      contact: editLead.contact ? editLead.contact._id : undefined,
-      lead: editLead._id,
-      organization: editLead.organization ? editLead.organization._id : undefined,
-      text: noteText,
-      user: this.props.userId,
-    };
+    let note: Note;
+    switch (this.props.modelType) {
+      case LEAD:
+        note = {
+          lead: this.props.model._id,
+          organization: this.props.model.organization ? this.props.model.organization._id : undefined,
```

Comment:
move this.props.model to const
const { model } = this.props;
and then remove redundant this.props part from all usages